src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift

```diff
@@ -41,7 +41,7 @@ class ExposureDetectionViewModel: CountdownTimerDelegate {
 				self?.riskProviderActivityState = $0
 
 				switch $0 {
-				case .downloading:
+				case .downloading, .riskManuallyRequested:
 					self?.setupForDownloadingState()
```

Comment:
I still see a very little, negligible problem with this. It will go from state downloading -> downloading in case of manual reload. Which will lead to a flickering in the UI. But I think its acceptable because the whole scenario is an edge case.

Reply:
Since the content is same in both states, I don't think there will be any flickering.